examples/planning/train.ipynb

```diff
@@ -123,7 +123,7 @@
     "# rasterisation and perturbation\n",
     "rasterizer = build_rasterizer(cfg, dm)\n",
     "perturbation = AckermanPerturbation(\n",
-    "        random_offset_generator=GaussianRandomGenerator(mean=np.array([0.0, 0.0]), std=np.array([1.0, np.pi / 6])),\n",
+    "        random_offset_generator=GaussianRandomGenerator(mean=np.array([0.0, 0.0, 0.]), std=np.array([1.0, 1.0, np.pi / 6])),\n",
```

Comment:
`mean=np.array([0.0, 0.0, 0.0]` or `mean=np.array([0., 0., 0.]` just for consistency :)
Change to `std=np.array([0.5, 1.5, np.pi / 6])`, lateral offset std =0.5m, longitudinal offset std =1.5m
Also add comments here saying the ordering are `lateral, longitudinal, yaw`